client/src/components/ConfigPanel.jsx

```
@@ -3008,68 +3149,124 @@ const ConfigPanel = ({
+                    </div>
+                  );
+                })}
+              </div>
+            )}
+
+            <div className={`mt-4 ${isPresetSelected ? 'opacity-50 pointer-events-none' : ''}`} aria-disabled={isPresetSelected ? 'true' : 'false'}>
+              <div className="rounded-xl border border-white/10 bg-black/20 px-3 py-2.5">
+                <label className="block text-[10px] font-bold uppercase tracking-widest text-white/40">
+                  Device
+                </label>
+                <select
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)